styles/Footer.module.scss

```diff
@@ -0,0 +1,315 @@
+    }
+
+    @include media-query(menu-break) {
+        padding: $grid * 2 $grid * 1.5 0px 0px;
+        &::before {
+            content: '';
+            position: absolute;
+            top: 18px;
+            left: -108px;
+            width: 2px;
+            height: 262px;
+            background-color: var(--color--dark-indigo);
+            transform: rotate(30deg);
+        }
+    }
+
+    &Heading {
+        font-weight: $weight--extrabold;
+        @include font-size(ml);
+        margin-top: 0;
+        margin-bottom: $grid;
+    }
+
+    &Location {
+        display: flex;
+        flex-direction: row;
+    }
+
+    &Icon {
+        display: block;
+        width: 32px;
+        margin-right: 5px;
+    }
+
+    &Information {
+        max-width: 90%;
+        // Set to 90% so as to be responsive without the address cropping the SVG icon
+
+        @include media-query(medium) {
+            width: 85%;
+        }
+
+        @include media-query(large) {
+            width: 90%;
+            max-width: 249px;
+        }
+    }
+
+    &Title {
+        @include font-size(xs);
+        font-weight: $weight--bold;
+        color: var(--color--dark-indigo);
+        margin-bottom: 0;
+    }
+
+    &Address {
+        @include font-size(xxs);
+        line-height: 19px;
+        color: var(--color--grey);
+        margin-bottom: $grid * 1.5;
+    }
+
+    &List {
+        display: grid;
+        grid-template-columns: 1fr;
+
+        @include media-query(medium) {
+            grid-template-columns: 1fr 1fr;
+        }
+    }
+}
+
+.getInTouch {
+    padding-top: $grid;
+
+    @include media-query(large) {
+        width: 50%;
+        padding: $grid * 2 $grid 0px $grid * 3;
+    }
+
+    @include media-query(menu-break) {
+        padding: $grid * 2 0px 0px $grid * 3.5;
+    }
+
+    &Title {
+        font-weight: $weight--extrabold;
+        @include font-size(ml);
+        margin-top: 0;
+    }
+
+    &Description {
+        @include font-size(s);
+
+        @include media-query(large) {
+            max-width: 474px;
+        }
+    }
+
+    &Contact {
+        display: flex;
+        flex-direction: row;
+        max-width: 340px;
+    }
+
+    &ContactInformation {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+    }
+
+    &EmailLink {
+        display: block;
+        line-height: 24px;
+        font-weight: $weight--bold;
+        color: var(--color--dark-indigo);
+        text-decoration-line: underline;
+
+        // This follows the designs as the @ doesn't cut through the line
+        text-underline-position: under;
+        // The extra spacing doesn't necessarily look as good.
+        // Maybe we should consider a pseudoelement underscore instead?
+    }
+
+    &PhoneLink {
+        display: block;
+        line-height: 24px;
+        font-weight: $weight--bold;
+        margin-top: 8px;
+    }
+
+    &Image {
+        max-width: 85px;
+        max-height: 85px;
```

Comment:
I don't think `max-` is needed here, we usually support a minimum of 320px in terms of device width, and this seems to work fine without these properties. 
**Question:** Was there a reason behind this, I tend to prefer media queries if needed.

Reply:
No, that wasn't needed - width is fine here :)